src/lib/components/infomessage/InfoMessage.component.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+import { Link, Text } from '../text/Text.component';
+import { Icon } from '../icon/Icon.component';
+import { defaultTheme } from '../../style/theme';
+import { useComputeBackgroundColor } from './InfoMessageUtils';
+
+type Props = {
+  title: string;
```

Comment:
i would make it possible to provide a ReactNode or a string here. It would allow use cases where we want to add icons or actions button towards the end of the title line.
```suggestion
  title: string | ReactNode;
```